src/views/faq/l10n.json

```diff
@@ -10,11 +10,14 @@
 	"faq.scratchExtensionsTitle":"Scratch Extensions",
     "faq.cloudDataTitle":"Cloud Variables",
     "faq.aboutScratchTitle":"What is Scratch, and what can I do with it?",
-    "faq.aboutScratchBody":"With the Scratch programming language and online community, you can create your own interactive stories, games, and animations -- and share your creations with others around the world. As young people create and share Scratch projects, they learn to think creatively, reason systematically, and work collaboratively. To learn more about Scratch, see the <a href=\"/about\">About Scratch</a> page.",
+    "faq.aboutScratchBody":"With the Scratch programming language and online community, you can create your own interactive stories, games, and animations -- and share your creations with others around the world. As young people create and share Scratch projects, they learn to think creatively, reason systematically, and work collaboratively. To learn more about Scratch, see the {aboutScratchLink} page.",
+        "faq.aboutScratchLinkText":"About Scratch",
     "faq.makeGameTitle":"How do I make a game or animation with Scratch?",
-    "faq.makeGameBody":"Check out the <a href=\"/ideas\">Ideas page</a> to see lots of ways to get started with Scratch",
+    "faq.makeGameBody":"Check out the {ideasLink} to see lots of ways to get started with Scratch",
+        "faq.ideasLinkText":"Ideas page",
     "faq.whoUsesScratchTitle":"Who uses Scratch?",
-    "faq.whoUsesScratchBody":"Scratch is used by people from all backgrounds, in all countries around the world, in all types of settings -- homes, schools, libraries, museums, and more. Scratch is designed especially for young people ages 8 to 16, but people of all ages create and share with Scratch. Younger children may want to try <a href=\"https://www.scratchjr.org/\">ScratchJr</a>, a simplified version of Scratch designed for ages 5 to 7.",
+    "faq.whoUsesScratchBody":"Scratch is used by people from all backgrounds, in all countries around the world, in all types of settings -- homes, schools, libraries, museums, and more. Scratch is designed especially for young people ages 8 to 16, but people of all ages create and share with Scratch. Younger children may want to try {scratchJrLink}, a simplified version of Scratch designed for ages 5 to 7.",
+        "faq.scratchJrLinkText":"ScratchJr",
```

Comment:
_ScratchJr_ shouldn't get translated, so it doesn't need to be in the l10n file.

Reply:
I anticipated this, but there were a bunch of other strings like "Scratch 2.0" or "Scratch Desktop" that I wasn't sure whether to translate. I left them all in in case you had comments. I'll remove this one, but do let me know if any other strings I added don't actually need to be there.